integrated_channels/sap_success_factors/migrations/0001_squashed_0022_auto_20200206_1046_squashed_0011_alter_sapsuccessfactorslearnerdatatransmissionaudit_enterprise_course_enrollment_id.py

```diff
@@ -76,5 +76,8 @@ class Migration(migrations.Migration):
                 ('created', models.DateTimeField(auto_now_add=True)),
                 ('credit_hours', models.FloatField(blank=True, null=True)),
             ],
+            options={
+                'db_table': 'sap_success_factors_sapsuccessfactorslearnerdatatransmission3ce5',
+            }
```

Comment:
Question: Why was this change necessary?

Reply:
If don't do this change the table name generated in postgresql doesn't match with the table name it generates in mysql. I believe it is because of the make table name length constraints.
The reason of pinning it is as we are using the table name while creating index in subsequent migration.
`0011_alter_sapsuccessfactorslearnerdatatransmissionaudit_index_together.py`